File: program51.c
#include<stdio.h>

int CountEvenDigits(int iNo)
{
    int iDigit = 0;
    int iEvenCnt=0;

    if(iNo==0)
    {
        return 1;
    }
    
    while(iNo!=0)
    {
        iDigit=iNo%10;
        iNo=iNo/10;

        if(iDigit%2==0)
        {
        iEvenCnt++;
        }
         
    }
    return iEvenCnt;
}

int main()
{
    int iValue = 0;
    int iRet = 0;
    printf("Enter a number:\n");
    scanf("%d",&iValue);

    iRet = CountEvenDigits(iValue);

    printf("Sum of Digits are:%d\n",iRet);
    return 0;
}

File: Program51.java
import java.util.*;

class NumberX{
    public int iValue=0;

    NumberX(int iNo1){
        this.iValue=iNo1;
        
    }

    public int Calculation(){
        int iSum=0;
        int iSquaresum=0;

        for(int iCnt=0;iCnt<iValue;iCnt++){
            iSum=iSum+iCnt;
        }

        for(int iCnt=0;iSquaresum<iValue;iCnt++){
            iSquaresum=iSquaresum+(iCnt*iCnt);
        }

        return iSquaresum-(iSum*iSum);
    }
}
public class Program51 {
    public static void main(String aee[]){
        Scanner sobj=new Scanner(System.in);

        System.out.println("enter number:");
        int iNumber=sobj.nextInt();

        NumberX obj=new NumberX(iNumber);
        int iRet=obj.Calculation();

        System.out.println("calculation is:"+iRet);
    }

}

